.travis.yml

```diff
@@ -1,22 +1,21 @@
 language: node_js
-
 node_js:
-  - 0.10
-
+- 0.1
 env:
   global:
-    - BROWSER_PROVIDER_READY_FILE=/tmp/sauce-connect-ready
-    - LOGS_DIR=/tmp/angular-router-build/logs
-    - SAUCE_USERNAME=angular-ci
-    - SAUCE_ACCESS_KEY=9b988f434ff8-fbca-8aa4-4ae3-35442987
-
+  - BROWSER_PROVIDER_READY_FILE=/tmp/sauce-connect-ready
+  - LOGS_DIR=/tmp/angular-router-build/logs
+  - SAUCE_USERNAME=angular-ci
+  - SAUCE_ACCESS_KEY=9b988f434ff8-fbca-8aa4-4ae3-35442987
+  - secure: iN9MYhd7RDn8JOQfKHVwKfOJLYSnEXYBt33xMhC+Q0xBNIFr516DiaWfQ0MWTsPmJThl7SJylDc+YdHX64SXEFEMsCSzksqVUddOUuc/JHlPG943ypkab1RgYCbVJ+qoppd5UXhsejSuivuY1Io4/i7mMFkP5LaM43F0ZPO67eo=
 install:
-  - mkdir -p $LOGS_DIR
-  - ./scripts/sauce_connect_setup.sh
-  - npm install
-  - npm install -g gulp
-  - npm install -g karma-cli
-  - ./scripts/wait_for_browser_provider.sh
-
+- mkdir -p $LOGS_DIR
+- "./scripts/sauce_connect_setup.sh"
+- npm install
+- npm install -g gulp
+- npm install -g karma-cli
+- "./scripts/wait_for_browser_provider.sh"
```

Comment:
Not sure to be honest. I haven't added them manually. I'm wondering if the Travis CLI tool added them.